python/pyspark/sql/tests/test_datasources.py

```diff
@@ -211,7 +211,7 @@ def test_checking_csv_header(self):
                 ]
             )
             df = (
-                self.spark.read.option("header", "true")
+                self.spark.read.option("header", "true").option("quote", None)
```

Comment:
`quote` option is null tolerant, and it would throw an exception if the char is more than 1. So this should verify the change here

Reply:
This test is also shared between Spark Classic and Connect.

Reply:
Thanks, yeah, the test with None will test it good. The problem with testing "True" is that most places tolerate case-insensitive "True" string (I ran into this bug because Delta CDC reader didn't, but I'm fixing it separately in Delta)